ensembl/htdocs/info/genome/variation/prediction/classification.html

```diff
@@ -197,7 +209,79 @@ <h2 id="classes">Variant classes</h2>
     </td>
     <td style="padding-left:0px;width:16px"><div><a href="/Homo_sapiens/StructuralVariation/Explore?sv=nsv1162620" target="_blank" title="See a structural variant example"><img src="/i/16/internal_link.png" alt="Link"/></a></div></td>
   </tr>
+  <tr>
+    <td style="padding:0px;margin:0px;;background-color:#000000;border-top:1px solid #FFF"></td>
```

Comment:
There are a lot of variant classes missing SO descriptions. I assume they are terms we have not previously supported that were in a SV import (@dglemos ?)
We need to check it they have valid SO terms and here are implications sa to whether other SVs should be described with this more specific term. Can you remove them from the docs for now?

Reply:
I missed this comment! 
These terms are valid SO terms, we will fix them for next release.